imgui/enums.pxd

```diff
@@ -217,91 +272,267 @@ cdef extern from "imgui.h":
-        # AcceptDragDropPayload() flags
+        #  AcceptDragDropPayload() flags                
         ImGuiDragDropFlags_AcceptBeforeDelivery         # AcceptDragDropPayload() will returns true even before the mouse button is released. You can then call IsDelivery() to test if the payload needs to be delivered.
         ImGuiDragDropFlags_AcceptNoDrawDefaultRect      # Do not draw the default highlight rectangle when hovering over target.
         ImGuiDragDropFlags_AcceptNoPreviewTooltip       # Request hiding the BeginDragDropSource tooltip from the BeginDragDropTarget site.
-        ImGuiDragDropFlags_AcceptPeekOnly = ImGuiDragDropFlags_AcceptBeforeDelivery | ImGuiDragDropFlags_AcceptNoDrawDefaultRect
-
+        ImGuiDragDropFlags_AcceptPeekOnly               = ImGuiDragDropFlags_AcceptBeforeDelivery | ImGuiDragDropFlags_AcceptNoDrawDefaultRect  # For peeking ahead and inspecting the payload before delivery.
+  
     ctypedef enum ImGuiDir_:
-        ImGuiDir_None
-        ImGuiDir_Left
-        ImGuiDir_Right
-        ImGuiDir_Up
-        ImGuiDir_Down
+        ImGuiDir_None   
+        ImGuiDir_Left   
+        ImGuiDir_Right  
+        ImGuiDir_Up     
+        ImGuiDir_Down   
         ImGuiDir_COUNT
+    
+    ctypedef enum ImGuiSortDirection_:
+        ImGuiSortDirection_None         #
+        ImGuiSortDirection_Ascending    # Ascending = 0->9, A->Z etc.
+        ImGuiSortDirection_Descending   # Descending = 9->0, Z->A etc.
 
     ctypedef enum ImGuiMouseCursor_:
-        ImGuiMouseCursor_None
-        ImGuiMouseCursor_Arrow
-        ImGuiMouseCursor_TextInput              # When hovering over InputText, etc.
-        ImGuiMouseCursor_ResizeAll
-        ImGuiMouseCursor_ResizeNS               # Unused
-        ImGuiMouseCursor_ResizeEW               # When hovering over a column
-        ImGuiMouseCursor_ResizeNESW             # Unused
-        ImGuiMouseCursor_ResizeNWSE             # When hovering over the bottom-right corner of a window
-        ImGuiMouseCursor_Count_
+        ImGuiMouseCursor_None 
+        ImGuiMouseCursor_Arrow 
+        ImGuiMouseCursor_TextInput,         # When hovering over InputText, etc.
+        ImGuiMouseCursor_ResizeAll,         # (Unused by Dear ImGui functions)
+        ImGuiMouseCursor_ResizeNS,          # When hovering over an horizontal border
+        ImGuiMouseCursor_ResizeEW,          # When hovering over a vertical border or a column
+        ImGuiMouseCursor_ResizeNESW,        # When hovering over the bottom-left corner of a window
+        ImGuiMouseCursor_ResizeNWSE,        # When hovering over the bottom-right corner of a window
+        ImGuiMouseCursor_Hand,              # (Unused by Dear ImGui functions. Use for e.g. hyperlinks)
+        ImGuiMouseCursor_NotAllowed,        # When hovering something with disallowed interaction. Usually a crossed circle.
+        ImGuiMouseCursor_COUNT
+
```

Comment:
This might conflict with #209 if that is merged before.

Reply:
Thanks! Stop me if I'm wrong but I think it should be ok, the only conflicting names are `MOUSE_CURSOR_NONE = enums.ImGuiMouseCursor_None` and `MOUSE_CURSOR_HAND = enums.ImGuiMouseCursor_Hand` and both versions are identical. Thus the conflict resolution should be trivial no ?

Reply:
Indeed.